cookiecutter.json

```diff
@@ -1,8 +1,16 @@
 {
-  "author_name": "Timothée Mazzucotelli",
-  "author_email": "timothee.mazzucotelli@gmail.com",
-  "repository_namespace": "Pawamoy",
+  "author_name": "David Mougeolle",
+  "author_email": "david.mougeolle@moodule.net",
```

Comment:
Let's drop the real names and use something generic like `Author Name` and `author.name@domain.com`

Reply:
yes!!
I should have split the commit into "json properties edit" & "personnal customization"...
I didn't know about atomicity at the time, sorry for dumping everything in the PR!!

Reply:
It's fine, don't worry :slightly_smiling_face: You're my first contributor, it's all I care about :smile: